src/app/api/graphql/queries/joined.ts

```diff
@@ -0,0 +1,25 @@
+import gql from "graphql-tag";
+
+export const JOINED_BY_USER = gql`
+    query JoinedByUser($eventId: ID!, $userId: ID!) {
+        joinedByUser(event_id: $eventId, user_id: $userId) {
+            _id
+            event_id
+            user_id
+            isJoined
+            createdAt
+            updatedAt
+        }
+    }
+`;
+
+export const JOINED_BY_USERS = gql`
+    query GetJoinedByUsers($eventId: ID) {
```

Comment:
Тут кажется можно сделать единообразнее: 
```suggestion
    query JoinedByUsers($eventId: ID) {
```

Reply:
Исправлено в следующем PR